includes/modules/column_left.php

```diff
@@ -25,7 +25,7 @@
 $column_width = (int)BOX_WIDTH_LEFT;
 foreach ($sideboxes as $sidebox) {
     $boxFile = (new SideboxFinder(new FileSystem))->sideboxPath($sidebox, $template_dir, true);
-    if (file_exists($boxFile)) {
+    if ($boxFile !== false && file_exists($boxFile)) {
```

Comment:
The check for `file_exists` is redundant, since the SideboxFinder has already verified that it exists if the return value isn't `(bool)false`.

Ditto for line 28 of the `column_right.php`, below.